mstranslator.py

```diff
@@ -64,17 +66,20 @@ def token(self):
 
 
 class Translator(object):
-    api_url = "https://api.microsofttranslator.com/v2/ajax.svc/"
+    api_url = "https://api.cognitive.microsofttranslator.com/"
 
     def __init__(self, subscription_key):
         self.auth = AccessToken(subscription_key)
 
     def make_url(self, action):
-        return self.api_url + action
+        return self.api_url + action + '?api-version=3.0'
 
-    def make_request(self, action, params=None):
+    def make_request(self, action, params=None, body=None, headers=None, isPost=True):
         url = self.make_url(action)
-        resp = requests.get(url, auth=self.auth, params=params)
+        if(isPost):
+            resp = requests.post(url, auth=self.auth, params=params, json=body, headers=headers)
+        else:
+            resp = requests.get(url, auth=self.auth, params=params, json=body, headers=headers)
```

Comment:
Does `json=body` make sense to GET request?

Reply:
You are right, already changed.